toolkit/authentication/src/main/java/com/arcgismaps/toolkit/authentication/OAuthWebView.kt

```diff
@@ -267,7 +267,8 @@ private class OAuthWebViewClient(
 
     /**
      * Completes the WebView OAuth challenge by either passing back a [redirectUrl] or an [exception].
-     * if [OAuthUserConfiguration.preferPrivateWebBrowserSession] is set to true, this method will clear the [webView]'s
+     *
+     * If [com.arcgismaps.httpcore.authentication.OAuthUserConfiguration.preferPrivateWebBrowserSession] is set to true, this method will clear the [webView]'s
```

Comment:
If the intention of this change is to have a qualified KDoc link, I would recommend to instead add the import to keep this doc unchanged:

```kotlin
import com.arcgismaps.httpcore.authentication.OAuthUserConfiguration
```


```suggestion
     *
     * If [OAuthUserConfiguration.preferPrivateWebBrowserSession] is set to true, this method will clear the [webView]'s
```

Reply:
Yeah the intention was to get rid of the warning: 
```
Cannot resolve symbol 'preferPrivateWebBrowserSession'
```
But if i instead add the import, it doesn't seem to work 😞 
The import shows as unused and the warning is still there.

Reply:
Try, gradle clean then build? I can get rid of the warnings with this and `[WebView]` as well.